Tests/ExamplesTest/expected_output/httpanalyzer_sanity.txt

```diff
@@ -6,7 +6,7 @@ STATS SUMMARY
 General stats
 -------------
 
-Sample time:                                      0.002 [Seconds]
+Sample time:                                      0.001 [Seconds]
```

Comment:
@seladb Is it possible to modify the test itself or to remove this lines? this section of the output seems to be very sensible to the execution environment... the figures change from time to time.

Reply:
I'm not sure I understand... the test already ignores this line:
https://github.com/jpcofr/PcapPlusPlus/blob/c5c63ded724abc1a3597cbfdd8039b720656d184/Tests/ExamplesTest/tests/test_httpanalyzer.py#L32

We don't need to change this value because the test ignores it